openedx/core/djangoapps/content/learning_sequences/api/tests/test_outlines.py

```diff
@@ -7,7 +7,7 @@
 import unittest
 
 from django.conf import settings
-from django.contrib.auth.models import AnonymousUser, User  # lint-amnesty, pylint: disable=imported-auth-user
+from django.contrib.auth.models import AnonymousUser  # lint-amnesty
```

Comment:
Since you fixed the import of User, you can also remove the # lint amnesty annotation